src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Ball ball = new Ball();
+
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/e1629f8a-fa64-4377-ac23-971c3a1324b8)